app/src/main/java/com/hayaisoftware/launcher/activities/SearchActivity.java

```diff
@@ -490,9 +507,12 @@ private void updateVisibleApps() {
     }
 
     private void sortApps() {
-        Collections.sort(mActivityInfos, mAlphabeticalOrderComparator);
```

Comment:
Keep this line. We always order alphabetically first, so in case of a "tie" in the selected comparison setting the app order stays alphabetical.